packages/edge-login-ui-rn/src/components/themed/EdgeTextFieldOutlined.js

```diff
@@ -46,16 +45,42 @@ const ANIMATION_STATES = {
   ERROR: 2
 }
 
-type InputOutlineProps = TextInputProps & {
+type Props = {|
```

Comment:
Perhaps export this type for use in the `edge-react-gui`?

Reply:
No, because we should really just share the entire component. Yes, building a common component library is on the roadmap, but there are a few puzzle pieces we need to move around before we can get to that point.